packages/codec/lib/read/index.ts

```diff
@@ -2,14 +2,15 @@ import * as storage from "./storage";
 import * as bytes from "./bytes";
 import * as stack from "./stack";
 import * as constant from "./constant";
-import * as Pointer from "@truffle/codec/types/pointer";
-import { EvmState } from "@truffle/codec/types/evm";
-import { DecoderRequest } from "@truffle/codec/types/request";
-import { Errors } from "@truffle/codec/format";
-
-export default function* read(pointer: Pointer.DataPointer, state: EvmState): Generator<DecoderRequest, Uint8Array, Uint8Array> {
-  switch(pointer.location) {
-
+import * as Pointer from "@truffle/codec/pointer";
+import { DecoderRequest } from "@truffle/codec/types";
+import * as Evm from "@truffle/codec/evm";
+
+export default function* read(
+  pointer: Pointer.DataPointer,
+  state: Evm.EvmState
+): Generator<DecoderRequest, Uint8Array, Uint8Array> {
+  switch (pointer.location) {
```

Comment:
Again, can't comment on things that haven't changed, but something's clearly gone wrong with the comments on this file; the comment on the `eventdata` case is meaningless.  Delete it I guess?